overlore/graphql/constants.py

```diff
@@ -0,0 +1,98 @@
+from enum import Enum
+
+
+class EventType(Enum):
+    ORDER_ACCEPTED_EVENT = "0x20e86edfa14c93309aa6559742e993d42d48507f3bf654a12d77a54f10f8945"
+    TRANSFER_EVENT = "0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9"
+    COMBAT_OUTCOME = "0x1736c207163ad481e2a196c0fb6394f90c66c2e2b52e0c03d4a077ac6cea918"
+
+
+class Queries(Enum):
+    PENDINGTRADES = """
+    tradeModels(where: {taker_idEQ: "0"}) {
+        edges {
+        node {
+            entity {
+            id
+            keys
+            updatedAt
+            }
+        }
+        }
+    }"""
+
+    PENDING_TRADES = """
```

Comment:
L11 and L24, PENDINGTRADES and PENDING_TRADES queries. Change the name of one of them?

Reply:
Deleted one of them that was unnecessary